apps/gimbtonomy/src/main.cpp

```diff
@@ -0,0 +1,77 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2018 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "CANMsg.h"
```

Comment:
can config and rover config are now always included in every file automatically. (rover config is done so trasitively thru mbed_config) which is also included automatically.